### ISSUE TITLE
Rest of Excercises

### ISSUE THREAD

**Issue by Adeelyousaf** [ge35tay/Event_Based_Vision#1]
Hi,

Kindly, if you could share details of rest of event based vision exercises. Any link to this course?

**Comment by ge35tay** [comment]
Yes, this is a course from Technical Tniversity Berlin, and here is the link to their course and all material about event based vision, hope you enjoy it :) https://sites.google.com/view/guillermogallego/teaching/event-based-robot-vision
